src/FlatSharp.Runtime/IFacadeTypeConverter.cs

```diff
@@ -0,0 +1,40 @@
+﻿/*
+ * Copyright 2021 James Courtney
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#nullable enable
```

Comment:
Do you think enabling `#nullable` project-wide is a good idea? If so, I could give it a go.

Reply:
I gave this a shot a year or so ago. The main thing I ran into was that it wasn't that useful at the time because I was trying to implement some interfaces that didn't themselves have the annotations, which reduced the usefulness of the whole tool chain. Maybe things have improved since then with .NET 5, etc. 

What would be cool would be to:
- Enable nullable in FlatSharp.Runtime (the generated C# makes liberal use of the runtime library for reusable chunks)
- Enable nullable in the Generated Code
- See what warnings pop out (maybe fail the compiler if it sees a nullability warning)

Nullable in some of the ancillary projects like the compiler / etc is interesting, but not something that would bite someone unexpectedly at runtime and throw a weird exception from code that they can't see.

Playing around with it a bit just now, things *do* look like they have gotten a bit better (or maybe I'm just smarter!) In any case, I'd welcome a PR applying project-wide nullability to the FlatSharp.Runtime as a starting point. Some files in there already have nullable turned on, so it may be not a huge amount of work.